src/epr/main.py

```diff
@@ -0,0 +1,554 @@
+            "--schema",
+            dest="schema",
+            action="store",
+            default=None,
+            help="Schema of the Event Receiver",
+        )
+        event_receiver_parser.add_argument(
+            "--fields",
+            dest="fields",
+            action="store",
+            default=None,
+            help="Fields of the Event Receiver",
+        )
+        event_receiver_group_parser = subparsers.add_parser(
+            "event-receiver-group", help="Event Receiver Group related options"
+        )
+        event_receiver_group_parser.add_argument(
+            "--id",
+            dest="id",
+            action="store",
+            default=None,
+            help="ID of the Event Receiver Group",
+        )
+        event_receiver_group_parser.add_argument(
+            "--name",
+            dest="name",
+            action="store",
+            default=None,
+            help="Name of the Event Receiver Group",
+        )
+        event_receiver_group_parser.add_argument(
+            "--type",
+            dest="type",
+            action="store",
+            default=None,
+            help="Type of the Event Receiver Group",
+        )
+        event_receiver_group_parser.add_argument(
+            "--version",
+            dest="version",
+            action="store",
+            default=None,
+            help="Version of the Event Receiver Group",
+        )
+        event_receiver_group_parser.add_argument(
+            "--description",
+            dest="description",
+            action="store",
+            default=None,
+            help="Description of the Event Receiver Group",
+        )
+        event_receiver_group_parser.add_argument(
+            "--event-receiver-ids",
+            dest="event_receiver_ids",
+            action="store",
+            default=None,
+            help="Event Receiver IDs of the Event Receiver Group",
+        )
+        event_receiver_group_parser.add_argument(
+            "--fields",
+            dest="fields",
+            action="store",
+            default=None,
+            help="Fields of the Event Receiver Group",
+        )
+        args = vars(parser.parse_args(sys.argv[2:]))
+        url = args["epr_url"]
+        token = args["epr_api_token"]
+        cfg = Config(url=url, token=token)
+
+        cfg.debug = args["debug"]
+        if args["subparser_name"] == "event":
+            event = Event()
+            event.id = args["id"]
+            event.name = args["name"]
+            event.version = args["version"]
+            event.release = args["release"]
+            event.platform_id = args["platform_id"]
+            event.package = args["package"]
+            event.description = args["description"]
+            event.payload = args["payload"]
+            event.success = args["success"]
+            event.event_receiver_id = args["event_receiver_id"]
+            cfg.events.append(event)
+            cfg.event_fields = args["fields"]
```

Comment:
Fields arg wasn't working for me, found that this (& equivalent for other resource commands) needs `.split(",")`. Otherwise the value remains a string whose characters are later joined in `client.py`'s `_fields = ",".join(fields)`.
```sh
❯ eprcli search event-receiver --id 01HWJZSMRM26ZSCQFKEA2MMHEN --fields 'name'
{'events': [], 'event_receivers': [{'errors': [{'message': 'Cannot query field "n" on type "EventReceiver".', 'locations': [{'line': 1, 'column': 79}]}, {'message': 'Cannot query field "a" on type "EventReceiver".', 'locations': [{'line
': 1, 'column': 81}]}, {'message': 'Cannot query field "m" on type "EventReceiver".', 'locations': [{'line': 1, 'column': 83}]}, {'message': 'Cannot query field "e" on type "EventReceiver".', 'locations': [{'line': 1, 'column': 85}]}]}], 'event_receiver_groups': []}
```

Reply:
fixed